tsl/src/continuous_aggs/create.c

```diff
@@ -383,18 +384,21 @@ cagg_add_trigger_hypertable(Oid relid, char *trigarg)
 		foreach (cell, ht->data_nodes)
 		{
 			HypertableDataNode *node = lfirst(cell);
-			StringInfo command = makeStringInfo();
-			appendStringInfo(command,
-							 "CREATE TRIGGER %s AFTER INSERT OR UPDATE OR DELETE ON %s.%s FOR EACH "
-							 "ROW EXECUTE FUNCTION %s.%s(%d, %d)",
-							 quote_identifier(CAGGINVAL_TRIGGER_NAME),
-							 quote_identifier(NameStr(ht->fd.schema_name)),
-							 quote_identifier(NameStr(ht->fd.table_name)),
-							 quote_identifier(INTERNAL_SCHEMA_NAME),
-							 quote_identifier(CAGG_INVALIDATION_TRIGGER),
-							 node->fd.node_hypertable_id, /* distributed member hypertable ID */
-							 node->fd.hypertable_id /* Access Node hypertable ID */);
-			cmd_descr_data[i].sql = command->data;
+			int ret1, ret2;
+			char trigarg1[NAMEDATALEN], trigarg2[NAMEDATALEN];
```

Comment:
You don't really need `NAMEDATALEN` bytes to hold an integer, only 12 bytes:
```suggestion
			char trigarg1[12], trigarg2[12];
```

Reply:
I was following the existing code:
https://github.com/timescale/timescaledb/blob/2.5.0/tsl/src/continuous_aggs/create.c#L1796

Should I change this?

Reply:
I think the same comment applies to the existing code you reference, but this is not part of your PR so would suggest to leave that for now.

Reply:
See [comment below](https://github.com/timescale/timescaledb/pull/3830#discussion_r755828443).